highlighters/main/test-data/parameter-to-global-alias.zsh

```diff
@@ -28,8 +28,8 @@
 # vim: ft=zsh sw=2 ts=2 et
 # -------------------------------------------------------------------------------------------------
 
-alias -g x=y
-local s=x
+alias -g global_alias_7cc33aff-18d4-48cd-8f97-38c272119a53=y
+local s=global_alias_7cc33aff-18d4-48cd-8f97-38c272119a53
```

Comment:
Could you please either document the magic number, or use a self-documenting name, or have the test skip when its assumptions don't hold (see `$skip_test`)?

Reply:
I thought using a GUID was self-expanatory, but I guess we can just use something like
`global_alias_which_hopefully_does_not_clash_with_an_external_command_name`

Reply:
The problem with a UUID is that it's not clear whether it was a fresh nonce or a well-known value.